contracts/Ovm_SpokePool.sol

```diff
@@ -171,5 +181,5 @@ contract Ovm_SpokePool is SpokePool {
     // Reserve storage slots for future versions of this base contract to add state variables without
     // affecting the storage layout of child contracts. Decrement the size of __gap whenever state variables
     // are added. This is at bottom of contract to make sure its always at the end of storage.
-    uint256[1000] private __gap;
+    uint256[998] private __gap;
```

Comment:
What is this change? Did we add variables to the OVM SpokePool that would move storage slots?

Reply:
My understanding is that since this class' inherited class was updated (the `CircleCCTPAdapter`) we'd need to modify this. In both cases, we didn't have any upgrade errors as outlined here: https://docs.openzeppelin.com/contracts/3.x/upgradeable#:~:text=You%20may%20notice%20that%20every,storage%20compatibility%20with%20existing%20deployments.

Reply:
If the variables are constant or immutable then they don't take slots and this shouldn't be decremented